src/components/reusable/pagetitle/pageTitle.scss

```diff
@@ -0,0 +1,55 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: block;
+}
+
+.page-title {
+  margin: 0;
+  &-wrapper {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0 16px;
```

Comment:
This left/right padding might need to be removed. In a grid or page layout, it will un-align the title from other elements.

Reply:
Its there in UX : 16px at right and left 

<img width="1440" alt="Screenshot 2024-03-13 at 7 33 24 PM" src="https://github.com/kyndryl-design-system/shidoka-applications/assets/140634142/cc46bb94-334b-4e52-927b-9d0643b029ab">

Reply:
I see, but that doesn't matter when it's used in practice in an actual page layout. Try it out and you'll see.